src/libdredd/src/mutation_replace_unary_operator.cc

```diff
@@ -163,13 +163,21 @@ std::string MutationReplaceUnaryOperator::GetFunctionName(
                                   ->getName(ast_context.getPrintingPolicy())
                                   .str());
 
+  // Since the optimised cases are the same for both op 0 and op 1, we only need
+  // to distinguish in the case op -1
+  if (optimise_mutations &&
+      MutationReplaceExpr::ExprIsEquivalentTo(*unary_operator_.getSubExpr(), -1,
+                                              ast_context)) {
+    result += "_minus_one";
+  }
+
```

Comment:
I notice the CI is failing due to redefinitions. I didn't look into it, but perhaps this isn't quite right.

Reply:
It seems that this may have been the issue. I've changed it to add the specifier in the three unique cases.